lib/PGcore.pm

```diff
@@ -812,67 +744,70 @@ sub tempDirectory {
 	return $self->{tempDirectory};
 }
 
-=head2 surePathToTmpFile
+=head1 Message channels
 
-	$path = surePathToTmpFile($path);
+There are three message channels
```

Comment:
Is there are 3rd? or just debug and warning?

Reply:
Or maybe the 3rd is internal_debug?  Just clarifying the comment.

Reply:
That is a good question.  That is how that was before.  I just moved it, and that is why it shows it changed.

Reply:
I changed it to `two` because it seems clear that is correct.